### PR TITLE
StorageObjectInUseProtection is GA

### PR DESCRIPTION
**Pull request by tengqm**
Update feature gate reference for 1.11

xref: kubernetes/kubernetes#62870

### CODE REVIEW

**Comment by mistyhacks** [comment]
The underlying structure of the docs repository has changed due to the recent
Hugo migration. Specifically, all the files previously under `docs/` are now in
`content/en/docs/`. I have rebased your PR without changing any of your work, so
that your changes are modifying the affected files in the new location, so this
PR is now reviewable again.

**If you were using Github's web UI rather than a local Git client, you can stop
reading and continue working in this PR.**

If you use the Git command-line client, you will need to make sure your local
branch is up to date with your fork. **If you have local commits that have not
been pushed to your fork, it may be complicated to get them back.** Do `git log`
and note their commit hashes for later.

Assuming your your fork's remote is called `origin`, hard-reset your local
branch to be the same as your fork's branch:

```bash
git fetch origin; git reset --hard origin/storage-protection-gate
```

**If you didn't have local commits, you're now ready to keep working on this PR
locally. You can stop reading now.**

If you did have local commits, cherry-pick them from oldest to newest. **For
each commit that needs to be cherry-picked:**

```bash
git cherry-pick <HASH>
```

You will need to resolve conflicts in these cherry-picked commits, which
involves moving your modified files into the correct new location. All the
content in `docs/` moved to `content/en/docs/`, so you can move your local files
accordingly:

- Do `git add` for each of the moved files
- Do `git mergetool` and accept the
deleted rather than modified file
- Continue the cherry-pick by doing `git cherry-pick --continue`


When you've cherry-picked all local commits back onto the local branch, push the
branch (with new commits) to your fork. You should not need to force the push.

File: content/en/docs/reference/feature-gates.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 title: Feature Gates
-content_template: templates/concept
 ---
 
-{{% capture overview %}}
+{% capture overview %}
 This page contains an overview of the various feature gates an administrator
 can specify on different Kubernetes components.
-{{% /capture %}}
+{% endcapture %}
```
Comment (mistyhacks) [inline content/en/docs/reference/feature-gates.md]: /assign @chenopis 

I'm not sure the right way to do this in Hugo yet.

File: content/en/docs/reference/feature-gates.md [hunk 2]
```diff
@@ -120,10 +120,9 @@ A *Beta* feature means:
   incompatible changes in subsequent releases. If you have multiple clusters
   that can be upgraded independently, you may be able to relax this restriction.
 
-{{< note >}}
 **Note:** Please do try *Beta* features and give feedback on them!
 After they exit beta, it may not be practical for us to make more changes.
-{{< /note >}}
+{: .note}
```
Comment (mistyhacks) [inline content/en/docs/reference/feature-gates.md]: Please revert this to use the Hugo short-code syntax (as shown in the red lines 123 and 126).

File: content/en/docs/reference/feature-gates.md [hunk 3]
```diff
@@ -221,6 +220,6 @@ Each feature gate is designed for enabling/disabling a specific feature:
   enables the usage of [`local`](/docs/concepts/storage/volumes/#local) volume
```
Comment (mistyhacks) [inline content/en/docs/reference/feature-gates.md]: Not your fault, but please also update all the links in this file to point to `/content/en/docs/` where they start with `/docs/`. This is due to the Hugo migration.

**Comment by mistyhacks** [comment]
I was wrong about the need to change /docs/ to /content/en/docs/ so you can ignore the request to do that. The things about Hugo shortcodes still need to be addressed.

File: content/en/docs/reference/feature-gates.md [hunk 4]
```diff
@@ -1,14 +1,13 @@
 ---
 title: Feature Gates
-content_template: templates/concept
```
Comment (mistyhacks) [inline content/en/docs/reference/feature-gates.md]: This needs to be reverted.

File: content/en/docs/reference/feature-gates.md (hunk 1 above)
Comment (mistyhacks) [reply content/en/docs/reference/feature-gates.md]: Verified that the way in red in line 9 is the new right way, so please revert these back to the Hugo way.

**Comment by mistyhacks** [comment]
/assign @chenopis 
Can you help me figure out why Netlify is not running?

**Comment by Bradamant3** [comment]
Because @mistyhacks already gave the real lgtm and bc we've figured out the Netlify problem isn't with this PR (preview looks fine):

/lgtm
/approve

**Pull request merged**